webapp/components/admin_console/admin_navbar_dropdown.jsx

```diff
@@ -71,6 +73,7 @@ export default class AdminNavbarDropdown extends React.Component {
                 teams.push(
                     <li key={'team_' + team.name}>
                         <Link
+                            id={Utils.createSafeId('swithTo'+team.name)}
```

Comment:
Good habit, the `swtichTo` is already safe and the `team.name` may be a react formatted text node which the `createSafeId` func knows how to handle.